src/main/java/meteordevelopment/meteorclient/commands/commands/FakePlayerCommand.java

```diff
@@ -28,13 +28,13 @@ public void build(LiteralArgumentBuilder<CommandSource> builder) {
         builder.then(literal("add")
             .executes(context -> {
                 FakePlayer fakePlayer = Modules.get().get(FakePlayer.class);
-                FakePlayerManager.add(fakePlayer.name.get(), fakePlayer.health.get(), fakePlayer.copyInv.get());
+                FakePlayerManager.add(fakePlayer.name.get(), fakePlayer.health.get(), fakePlayer.copyInv.get(), Modules.get().get(FakePlayer.class).allowDamage.get());
                 return SINGLE_SUCCESS;
             })
             .then(argument("name", StringArgumentType.word())
                 .executes(context -> {
                     FakePlayer fakePlayer = Modules.get().get(FakePlayer.class);
-                    FakePlayerManager.add(StringArgumentType.getString(context, "name"), fakePlayer.health.get(), fakePlayer.copyInv.get());
+                    FakePlayerManager.add(StringArgumentType.getString(context, "name"), fakePlayer.health.get(), fakePlayer.copyInv.get(), Modules.get().get(FakePlayer.class).allowDamage.get());
```

Comment:
you do realize fakePlayer is literally a local variable here you dont need to reget it

Reply:
Didn't see that, thanks for pointing it out!